### PR TITLE
WIP - pdcli-oncall (OSD - 8119)

### PR DESCRIPTION
**Pull request by MitaliBhalla**

### CODE REVIEW

File: cmd/pdcli/oncall/cmd.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	Args:  cobra.NoArgs,
+	//RunE:  OnCall(),
+}
+
+//func init() {
+
+//Cmd.Flags().StringVar(&OncallArgs.apiKey, "key", "", "Access API key/token generated from "+APIKeyURL+"\nUse this option to overwrite the existing API key.")
+//}
+
+func OnCall() {
```
Comment (supreeth7) [inline cmd/pdcli/oncall/cmd.go:40]: It's a good practice to push errors upstream, so would make sense if `OnCall` returns an error.
I don't think OnCall needs to be exportable.

Comment (supreeth7) [reply cmd/pdcli/oncall/cmd.go:40]: As `OnCall` is the main command handler, essential cobra params are missing I suppose.

Comment (MitaliBhalla) [reply cmd/pdcli/oncall/cmd.go:40]: > As `OnCall` is the main command handler, essential cobra params are missing I suppose.

Added

Comment (MitaliBhalla) [reply cmd/pdcli/oncall/cmd.go:40]: > It's a good practice to push errors upstream, so would make sense if `OnCall` returns an error.
> I don't think OnCall needs to be exportable.

added error in return part of the function

**Pull request closed without merging**